custom-connectors/AzureADPhoneMethods/apiDefinition.swagger.json

```diff
@@ -15,11 +15,23 @@
     "application/json"
   ],
   "paths": {
-    "/beta/users/{userID}/authentication/phoneMethods": {
+    "/beta/users/{userID}/authentication/fido2Methods": {
       "get": {
+        "summary": "List fido2Methods",
```

Comment:
@BakkerJan Summary values are visible to public, please make them all grammar-mistake free.
In this case, It can be `"List fido2 Methods"`